From c7256ddeaa19b1045e5c1118636d3233ae7b4862 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Fri, 5 Jan 2018 17:41:01 -0800
Subject: [PATCH 07/10] dma: xilinx: dpdma: correct the error message

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 76ee212583820856aca5bbad818452a7a8ef8621)
---
 drivers/dma/xilinx/xilinx_dpdma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/xilinx/xilinx_dpdma.c b/drivers/dma/xilinx/xilinx_dpdma.c
index 2944e30..f9b0e94 100644
--- a/drivers/dma/xilinx/xilinx_dpdma.c
+++ b/drivers/dma/xilinx/xilinx_dpdma.c
@@ -2240,7 +2240,7 @@ static int xilinx_dpdma_probe(struct platform_device *pdev)
 
 	ret = dma_async_device_register(ddev);
 	if (ret) {
-		dev_err(xdev->dev, "failed to enable the axi clock\n");
+		dev_err(xdev->dev, "failed to register the dma device\n");
 		goto error_dma_async;
 	}
 
-- 
1.8.5.2.233.g932f7e4

